storage/postgreskv/alternateclient_test.go

```diff
@@ -13,9 +15,21 @@ import (
 	"storj.io/storj/storage/testsuite"
 )
 
+var (
+	doAltTests = flag.Bool("test-postgreskv-alt", false, "Run the KeyValueStore tests against the alternate PG implementation")
```

Comment:
```
postgresTestAddr = flag.String("postgres-addr", os.Getenv("STORJ_POSTGRES_TEST"), "Run the KeyValueStore tests against the alternate PG implementation")
```

Then in tests you can check for `if *postgresTestAddr == "" { t.Skip ...`

Reply:
You mean, to allow for supplying a different postgresql database when running the alt-tests than the one used for the main postgres tests? That doesn't seem like a very pressing need.

Reply:
I mean is there a reason you are skipping the alternate tests? Although now thinking about, yeah.. it might be fine.

But, yeah, just use the same database. But, for the other flag you can use `os.Getenv("STORJ_POSTGRES_TEST")` as the default parameter to make the code shorter.

Reply:
Yeah, I don't expect the "alternate client" to be around for long. I really just want it to be in the history in case it's needed for future reference. And I don't want to clutter up real, regular test results with tests of that throwaway code.